pkgs/build-support/fetchgithub/default.nix

```diff
@@ -53,16 +57,19 @@ let
   fetcherArgs = (if useFetchGit
     then {
       inherit rev deepClone fetchSubmodules sparseCheckout; url = gitRepoUrl;
+      passthru = newPassthru;
     } // lib.optionalAttrs (leaveDotGit != null) { inherit leaveDotGit; }
     else {
       url = "${baseUrl}/archive/${rev}.tar.gz";
 
       passthru = {
         inherit gitRepoUrl;
-      };
+      } // newPassthru;
     }
   ) // privateAttrs // passthruAttrs // { inherit name; };
 in
 
-fetcher fetcherArgs // { meta = newMeta; inherit rev owner repo; }
+(fetcher fetcherArgs).overrideAttrs (finalAttrs: previousAttrs: {
```

Comment:
I think we can omit previousAttrs and replace finalAttrs with _. Maybe we can even omit both.